config/deployment/test/dispatch.yaml

```diff
@@ -19,6 +19,8 @@ dispatch:
     service: pepper-api-spec
   - url: "pepper-test.datadonationplatform.org/*"
     service: pepper-backend
+  - url: "dsm-dev.datadonationplatform.org/ddp/*"
+    service: study-manager-backend
```

Comment:
You sure you want it to be dsm-dev in test, staging and dev?

Reply:
OMG